src/app/(tabs)/home.tsx

```
@@ -193,35 +405,239 @@ export default function HomeScreen() {
+
+  const commitMonthChange = useCallback((delta: 1 | -1) => {
+    setViewedMonth(
+      (month) => new Date(month.getFullYear(), month.getMonth() + delta, 1),
+    );
+  }, []);
+
+  // dragX를 여기서 바로 0으로 되돌리면 패널 내용(previousMonthWeeks 등)이 새
+  // viewedMonth로 다시 그려지기 전에 위치부터 가운데로 스냅돼 한 프레임 깜빡인다.
+  // useLayoutEffect로 재렌더가 커밋된 뒤에 리셋해서 내용과 위치가 같이 바뀌게 한다.
+  useLayoutEffect(() => {
+    dragX.value = 0;
```

ESLint

[error] Error: This value cannot be modified

Modifying a value returned from a hook is not allowed. Consider moving the modification into the hook where the value is constructed.

/home/jailuser/git/src/app/(tabs)/home.tsx:437:5
  435 |   // useLayoutEffect로 재렌더가 커밋된 뒤에 리셋해서 내용과 위치가 같이 바뀌게 한다.
  436 |   useLayoutEffect(() => {
> 437 |     dragX.value = 0;
      |     ^^^^^ `dragX` cannot be modified
  438 |   }, [viewedMonth, dragX]);
  439 |
  440 |   function goToPreviousMonth() {

(react-hooks/immutability)